src/stepper.cpp

```diff
@@ -182,6 +243,10 @@ void Stepper::SetTargetPosition(float value) {
     fas_stepper->moveTo(pos_steps);
 }
 
+void Stepper::SetTargetVelocity(float value) {
```

Comment:
i left it unimplemented so if someone would call it without providing implementation, code will not compile